Makefile

```diff
@@ -114,6 +116,22 @@ PLUGIN_PROTOS= \
 	google/protobuf/descriptor.proto \
 	swift-options.proto
 
+XCODE_BUILD_EXTRAS =
+# Invoke make with XCODE_SKIP_OPTIMIZER=1 to suppress the optimizer when
+# building the Xcode projects. For Release builds, this is a non trivial speed
+# up for compilation
+XCODE_SKIP_OPTIMIZER=0
+ifeq "$(XCODE_SKIP_OPTIMIZER)" "1"
+  XCODE_BUILD_EXTRAS += SWIFT_OPTIMIZATION_LEVEL=-Onone
+endif
```

Comment:
We could flip option name and default here so we normally skip the optimizer to speed things up.